src/main/java/core/basesyntax/service/files/FileWriterCsvImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.service.files;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class FileWriterCsvImpl implements FileWriterCsv {
+    @Override
+    public void write(String data, String filePath) {
+        try (BufferedWriter bw = new BufferedWriter(new FileWriter(filePath))) {
+            bw.write(data);
+            bw.newLine();
+        } catch (IOException e) {
+            throw new RuntimeException("can't write data to file", e);
```

Comment:
Good job on using try-with-resources and rethrowing a RuntimeException with an informative message when handling IOException.